scala-package/assembly/src/main/assembly/javadoc.xml

```diff
@@ -9,7 +9,7 @@
   <fileSets>
     <fileSet>
       <directory>${rootdir}/core/target/site/scaladocs</directory>
-      <outputDirectory>/</outputDirectory>
+      <outputDirectory>.</outputDirectory>
```

Comment:
How will this provide better result?

Reply:
It throws a warning because it causes problems on Windows